App.js

```diff
@@ -1,25 +1,66 @@
 import React from 'react'
-import styled from 'styled-components/native'
 
-const Container = styled.View`
-  flex: 1;
-  background-color: papayawhip;
-  justify-content: center;
-  align-items: center;
-`
+import { NavigationContainer } from '@react-navigation/native';
+import { createStackNavigator } from '@react-navigation/stack';
 
-const Title = styled.Text`
-  font-size: 24px;
-  color: palevioletred;
-`
+import { Popular } from './screens/Popular'
+import { Homescreen } from './screens/Homescreen'
+import { TopRated } from './screens/TopRated'
+import { Upcoming } from './screens/Upcoming'
+import { Detail } from './screens/Detail'
 
 const App = () => {
+
+  const Stack = createStackNavigator();
+
   return (
-    <Container>
-      <Title>This is your cool app!</Title>
-      <Title>Go to App.js and start coding</Title>
-      <Title>💅💅💅</Title>
-    </Container>
+    <>
+    <NavigationContainer>
+      <Stack.Navigator initialRouteName="Home">
+        <Stack.Screen name="Home" component={Homescreen} 
+        options={{
+          headerStyle: {
+            backgroundColor: 'black',
+          },
+          headerTintColor: '#fff',
+        }}
+        />
+        <Stack.Screen name="Popular" component={Popular}
+        options={{
+          headerStyle: {
+            backgroundColor: 'black',
+          },
+          headerTintColor: '#fff',
+        }}
+        />
+        <Stack.Screen name="TopRated" component={TopRated} 
+        options={{
+          headerStyle: {
+            backgroundColor: 'black',
+          },
+          headerTintColor: '#fff',
+        }}
+        />
+        <Stack.Screen name="Upcoming" component={Upcoming} 
+        options={{
+          headerStyle: {
+            backgroundColor: 'black',
+          },
+          headerTintColor: '#fff',
+        }}
+        />
+        <Stack.Screen name="Detail" component={Detail}
```

Comment:
I was not able to open the detail page of any of the movies 😭

Reply:
@KarolCislowski - I have now fixed the problem!